src/scenes/Home/index.js

```diff
@@ -24,16 +25,27 @@ class Home extends Component {
     super(props)
 
     this.state = {
-      searchText: ''
+      searchText: '',
+      nodes: null
     }
   }
 
   handleSearchChange (value) {
-    this.setState({ searchText: value })
+    const self = this
+
+    self.setState({ searchText: value })
+
+    if (value !== '') {
+      search.simpleQuery(value, nodes => {
```

Comment:
You could store your callback in a variable and call `.bind(this)` on it :+1:

Reply:
An other solution could be to use an arrow function, otherwise it may be better to use an hoc and redux to keep the component stateless ?

Reply:
We weren't talking about the same thing, my suggestion is about using arrow function here:
```js
const handleSearchChange = (value) => {
     this.setState({ searchText: value })
```
to prevent 'this binding' here:
```js
<Searchbar text={searchText}
            search={this.handleSearchChange.bind(this)}
            style={{ marginTop: '3rem', marginBottom: '2rem' }}/>
```

Reply:
Being able to do that is an experimental feature plugin in babel: "transform-class-properties" (Source: https://stackoverflow.com/a/31362350)
However, I am using `create-react-app` which doesn't allow edition to the babel config at the moment (But is on the [roadmap for version 2](https://github.com/facebook/create-react-app/issues/3815) with babel macros)

So:
* ⚙️ Should I eject from `create-react-app` and deal with all the configuration myself ?
* ⛔️ Should I keep using `.bind(this)` and wait for the next version ?

Reply:
Oh, next version of create-react-app [should be on friday](https://github.com/facebook/create-react-app/issues/3815#issuecomment-368322167), so I will fix this then